### PR TITLE
Bump winapi to 0.3

### PR DESCRIPTION
**Pull request by Eijebong**
I had to drop all those `#[derive(Debug)]` because winapi types don't
implement Debug anymore.

### CODE REVIEW

File: src/bitmap_render_target.rs [hunk 1]
```diff
@@ -93,11 +96,11 @@ impl BitmapRenderTarget {
...
         unsafe {
             let memory_dc = self.get_memory_dc();
-            let mut bitmap: winapi::BITMAP = zeroed();
-            let ret = gdi32::GetObjectW(gdi32::GetCurrentObject(memory_dc, winapi::OBJ_BITMAP),
-                                        size_of::<winapi::BITMAP>() as i32,
-                                        &mut bitmap as *mut _ as *mut winapi::c_void);
-            assert!(ret == size_of::<winapi::BITMAP>() as i32);
+            let mut bitmap: BITMAP = zeroed();
+            let ret = GetObjectW(GetCurrentObject(memory_dc, OBJ_BITMAP),
+                                        size_of::<BITMAP>() as i32,
```
Comment (emilio) [inline src/bitmap_render_target.rs]: nit: Indentation is off here.

**Review by emilio — APPROVED** [review]

**Comment by Eijebong** [comment]
Test failure is legit, I'll fix it

**Comment by emilio** [comment]
Thanks!

**Pull request merged**